src/Repository/ImageRepository.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Image;
+use App\Entity\User;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\AbstractQuery;
+
+/**
+ * @method user|null find($id, $lockMode = null, $lockVersion = null)
+ * @method user|null findOneBy(array $criteria, array $orderBy = null)
```

Comment:
Annotations à supprimer

Reply:
Salut, 
Dans tous les repositories ? 
Car c'est ajouté par PhpStorm pour préciser les méthodes find etc ...
Merci.

Reply:
certes masi ta méthode ne renvoie pas un User mais une Image. Soit tu les enlèves, soit tu mets les bonnes annotations ;)

Reply:
Done and for Video too.